docs/sources/enterprise/license/license-restrictions.md

```diff
@@ -6,66 +6,175 @@ aliases = ["/docs/grafana/latest/enterprise/license-restrictions"]
 weight = 300
 +++
 
-# License restrictions
+# Understanding Grafana Enterprise licensing
 
-Enterprise licenses are limited by the number of active users, the license expiration date, and the URL of the Grafana instance.
+When you become a Grafana Enterprise customer, you receive a license that governs your use of Grafana Enterprise.
 
-## User limits
+You either have:
 
-Users are limited by the number of active users and the number of concurrent sessions for a given account.
+- tiered licensing, or
+- combined licensing
+
+To understand which license type you have, refer to [Determine the number of active users for each licensed role](determine-the-number-of-active-users-for-each-licensed role).
```

Comment:
This link displays as `[Determine the number of active users for each licensed role](determine-the-number-of-active-users-for-each-licensed role)` in output.

Reply:
fixed.